trees/java/pre_in_post_traversal.java

```diff
@@ -0,0 +1,75 @@
+class Node{  
+    int value;  
+    Node left;  
+    Node right;  
+
+    public Node(int value) {  
+        this.value = value;  
+    }  
+}
+
+public class pre_in_post_traversal{
+
+    public static void preOrder(Node node){
+        if (node == null) return;
+
+        System.out.print(node.value + ", ");
+        preOrder(node.left);
+        preOrder(node.right);
+    }
+
+    public static void inOrder(Node node){
+        if(node == null) return;
+
+        inOrder(node.left);
+        System.out.print(node.value + ", ");
+        inOrder(node.right);
+    }
+
+    public static void postOrder(Node node){
+        if(node == null) return;
+        postOrder(node.left);
+        postOrder(node.right);
+        System.out.print(node.value + ", ");
+    }
+
+
+    public static void main(String [] args){
+        Node root = new Node(4);
+        root.left = new Node(2);
+        root.right = new Node(7);
+        root.left.right = new Node(6);
+        root.left.left = new Node(1);
+        root.right.right = new Node(9);
+
+        System.out.print("pre order: ");
+        preOrder(root);
+
+        System.out.print("\n\nin order: ");
```

Comment:
This is why py lint test is failing ( reference issue #179 ), it's because of `codespell`, which reports 

```
./trees/java/pre_in_post_traversal.java:48: nin ==> inn, min, bin, nine
```
it is considering the newline character ('\n') to be a part of work "in" which is generating this error.

Although it is not a bug, but still if someone want to fix it then a simple fix would be to separate newline characters with a new print statement. 

maybe something like this should work

```java
System.out.print("\n\n");
System.out.print("in order: ");
```

Reply:
thanks @Arsenic-ATG  for the suggestion
I'm changing it now

Reply:
now it pass the test 
thanks @Arsenic-ATG